jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -576,6 +564,13 @@ int jbmc_parse_optionst::doit()
         (void)verifier();
       }
 
+      if(options.get_bool_option("symex-driven-lazy-loading"))
+      {
+        // We can only output these after goto-symex has run.
+        (void)show_loaded_symbols(*goto_model_ptr);
+        (void)show_loaded_functions(*goto_model_ptr);
```

Comment:
I don't understand this bit of code: the comment says that we cannot do any of this output, but then invokes two output functions, which seems like a contradiction. And then why does it do any output at all here, where has this been requested?

Reply:
It says we needed to wait until after symex, which we just ran (the `verifier()` call above)

Reply:
(normally of course we can `--show-goto-functions` before symex, but not when symex itself is deciding what gets loaded)

Reply:
Ok, but why are we doing this output at all?